test/jdk/ProblemList.txt

```diff
@@ -465,6 +465,9 @@ java/awt/KeyboardFocusmanager/ConsumeNextMnemonicKeyTypedTest/ConsumeNextMnemoni
 java/awt/Window/GetScreenLocation/GetScreenLocationTest.java 8225787 linux-x64
 java/awt/Dialog/MakeWindowAlwaysOnTop/MakeWindowAlwaysOnTop.java 8266243 macosx-aarch64
 java/awt/dnd/BadSerializationTest/BadSerializationTest.java 8277817 linux-x64,windows-x64
+java/awt/dnd/DragSourceMotionListenerTest.java 8225131 windows-all
```

Comment:
`DragSourceMotionListenerTest` passes for me on Windows. Does it need to be problem-listed?

Reply:
It passes on windows. I'm checking with the updated changes and running it multiple times to check for any intermittent issues.

Reply:
The test fails on CI on windows when run multiple times. Hence I'll keep it problemlisted.